app/models/actor.rb

```diff
@@ -0,0 +1,11 @@
+class Actor < ApplicationRecord
+  has_many :movie_actors
+  has_many :movies, through: :movie_actors
+
+  validates :name, presence: true 
+  validates :age, presence: true, numericality: true 
+
+  def self.average_age
+    average(:age).round(2)
```

Comment:
This US was actually asking to see all the given *movie instance's* actors rather than all actors in the database. So, a better spot for this would be in the Movie model where we could call `actors.average(:age).round(2)`